week04/index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    
+    <title>가위바위보</title>
+</head>
+<body>
+    <div class="top_end"></div>
+    <div class="background">
+        <div class="top">
+            <div class="score">
+                <div id="humanscore">0</div>
+                <div id="vs"> : </div>
+                <div id="computerscore">0</div>
+            </div>
```

Comment:
불필요한 태그입니다. 태그를 잘개 쪼개지 않고도 JavaScript에서 템플릿 리터럴을 사용하여 값을 변경할 수 있습니다.
```suggestion
            <div class="score"></div>
```